src/it/parent-3x/invoker.properties

```diff
@@ -1,3 +1,3 @@
 # install, not verify, because we want to check the artifact as we would be about to deploy it
 # release.skipTests normally set in jenkins-release profile since release:perform would do the tests
-invoker.goals=-Pjenkins-release -Drelease.skipTests=false clean install
+invoker.goals=-Pjenkins-release -Drelease.skipTests=false clean install hpi:run
```

Comment:
I am not sure it is a good idea to invoke hpi:run in the test. What is an expected behavior?